js/TabEnhancements/TabEnhancements.js

```diff
@@ -18,47 +18,22 @@ class TabEnhancements {
         // Make sure tab panes are at least as high as the tab list (cms legacy tabs list)
         this.$html.find('.tabs > .tabs__content > .tab__pane').css('min-height', this.$html.find('.tabs__list').height());
 
-        // Switch the first active tab to use <main> with the skip target
-        this.changeActiveTabElementToMain(this.$html.find('.tab__pane.is-active'));
+        // Remove skip target IDs set in markup
```

Comment:
These changes are already in develop, so I'm confused why these are showing as a diff?